mrq/job.py

```diff
@@ -17,6 +17,7 @@
 import encodings
 import copy_reg
 from . import context
+from mrq.context import get_current_config
```

Comment:
context is imported right before, could you use context. get_current_config instead to avoid the double import?